.devcontainer/conf/nginx/nginx.conf

```diff
@@ -0,0 +1,42 @@
+user nginx;
+worker_processes  4;
+daemon off;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+    access_log  /var/log/nginx/access.log;
+    sendfile        on;
+    keepalive_timeout  65;
+
```

Comment:
I guess there is a solution to keep only the `server { ... }` part of the config.